pkg/codegen/python/doc.go

```diff
@@ -52,7 +52,7 @@ func (d DocLanguageHelper) GetDocLinkForResourceType(pkg *schema.Package, modNam
 	case pkg.Name == "" && modName != "":
 		path = modName
 	case pkg.Name != "" && modName == "":
-		path = pkg.Name
+		path = fmt.Sprintf("pulumi_%s", pkg.Name)
 	}
 	return fmt.Sprintf("/docs/reference/pkg/python/%s/#%s", path, typeName)
```

Comment:
This url seems to work fine as far as going to the right page. Though it seems the anchor (which is the typeName in this case) should also be prefixed with pulumi_pkgName as follows: `pulumi_docker.{typeName}` so it would look like `https://www.pulumi.com/docs/reference/pkg/python/pulumi_docker/#pulumi_docker.Container` This is the link that is generated on the page when I click the link icon for the container resource.

Reply:
Yep great catch! It looks like this was never right. I'll fix this up.